app/src/main/res/values/styles.xml

```diff
@@ -522,4 +523,36 @@
         <item name="android:background">@color/theme_divider</item>
     </style>
 
+    <!--..........................New Redesign...........................-->
```

Comment:
What is the rationale behind this naming? Is the design actually named "redesign"? Which we're introducing gradually?

Reply:
yes, the new onboarding flow comes with the redesign of the UI experience. new styles etc will be added and I do not want to mix things up. this will most likely change in future as I get time for proper tasks that refactor the structure here.